spring-kafka/src/main/java/org/springframework/kafka/listener/DeadLetterPublishingRecoverer.java

```diff
@@ -79,6 +87,14 @@
 
 	private Duration partitionInfoTimeout = Duration.ofSeconds(FIVE);
 
+	private boolean replaceOriginalHeaders = true;
+
+	private boolean failIfSendResultIsError = false;
+
+	private boolean throwIfNoDestinationReturned = false;
+
+	private long waitForSendResultTimeout = 30000L;
```

Comment:
Magic number  (see `partitionInfoTimeout` above) - also, we prefer `Duration` for these timeouts; see `setPartitionInfoTimeout()`.

Reply:
Yeah, me too, the get method from the Future interface takes a long. I'll do the conversion.